tests/frontier/precompiles/test_precompiles.py

```diff
@@ -109,3 +115,107 @@ def test_precompiles(
     post = {account: Account(storage={0: "0x00" if precompile_exists else "0x01"})}
 
     state_test(env=env, pre=pre, post=post, tx=tx)
+
+
+def precompile_addresses_in_successor(fork: Fork) -> Iterator[Tuple[str, bool]]:
+    """
+    Yield the addresses of precompiled contracts and whether they existed in the parent fork.
+
+    Args:
+        fork (Fork): The transition fork instance containing precompiled contract information.
+
+    Yields:
+        Iterator[Tuple[str, bool]]: A tuple containing the address in hexadecimal format and a
+            boolean indicating whether the address has existed in the predecessor.
+
+    """
+    predecessor_precompiles = [
+        hex(int.from_bytes(address, byteorder="big"))
+        for address in get_transition_fork_predecessor(fork).precompiles()
```

Comment:
Is there an issue with `pytest.mark.parametrize_by_fork` that requires a string or something else behind this?

Reply:
This just returns `Cancun` for `CancunToPragueAtTime15k`, it was introduced in https://github.com/ethereum/execution-spec-tests/pull/1313 where @spencer-tb fixed up how `state_test` gets named in the context of a transition test.

But if we just check a hard-coded range as @jochem-brouwer suggests above, then we don't need this "clever" logic would suggest converting it to a regular fork `state_test`.